examples/draft-pad/app/api/gensx/route.ts

```diff
@@ -66,24 +66,28 @@ export async function POST(request: NextRequest) {
         environment,
       });
     } else {
+      console.log("Using local dev server");
```

Comment:
maybe worth ditching the console.log statements in this file?

Reply:
shoot, yeah just merged before seeing this but will remove in my next pr